h2o-py/h2o/frame.py

```diff
@@ -1093,11 +1093,6 @@ def __getitem__(self, item):
         fr._ex._cache.types = new_types
         return fr
 
-    # def __getattr__(self, key):
```

Comment:
Just naive idea following ^^: we can say that rule is `frame.COL_<actual colname>`

But what if colname would contain string like "average+age+ca" (this is from census dataset)?